### PR TITLE
feat: Added dev mode socket urls

### PR DESCRIPTION
**Pull request by Danwakeem**
## Description
Added dev mode socket urls so we can use them in serverless framework.

### CODE REVIEW

**Comment by Danwakeem** [comment]
This will be used in serverless/serverless#11782 😎

**Comment by Danwakeem** [comment]
I am guessing that might have just been some windows test flakiness 🤔

**Review by medikoo — COMMENTED** [review]
@Danwakeem I've added some debug logs to tests, so if it happens again we will have more information, can you rebase against main?

**Comment by Danwakeem** [comment]
@medikoo the windows step didn't fail that time 😢 but I guess those logs will be helpful if the test flakes in the future 👍

File: lib/auth/urls.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const config = new Map([
       backend: 'https://core.serverless-dev.com',
       integrationsBackend: 'https://integrations.core.serverless-dev.com',
       frontend: 'https://console.serverless-dev.com',
+      devMode: 'wss://dev.socket.core.serverless-dev.com',
```
Comment (medikoo) [inline lib/auth/urls.js:10]: Maybe let's name it `devModeFeed` ?

Comment (Danwakeem) [reply lib/auth/urls.js]: That sounds reasonable to me 👍 🚀

**Review by medikoo — APPROVED** [review]
👍

**Pull request merged**